src/images/falseColourPanel.component.tsx

```diff
@@ -99,127 +208,175 @@ const FalseColourPanel = (props: FalseColourPanelProps) => {
     }
     setEnabled(checked);
   };
+  const handleReverseColour = (
+    event: React.ChangeEvent<HTMLInputElement>,
+    checked: boolean
+  ) => {
+    if (selectColourMap !== '') {
+      if (!checked) {
+        changeColourMap(selectColourMap);
+      } else {
+        changeColourMap(`${selectColourMap}_r`);
+      }
+    }
+    setReverseColour(checked);
+  };
+  function filterNamesWithSuffixR(
```

Comment:
This function can be extracted outside the component - a slight performance optimisation